src/Symfony/Component/HttpKernel/DataCollector/Util/ValueExporter.php

```diff
@@ -75,4 +79,11 @@ public function exportValue($value, $depth = 1, $deep = false)
 
         return (string) $value;
     }
+
+    private function getClassNameFromIncomplete(\__PHP_Incomplete_Class $value)
+    {
+        $array = new \ArrayObject($value);
+
+        return $array['__PHP_Incomplete_Class_Name'];
```

Comment:
yeah, __PHP_Incomplete_Class is really weird (the fact that ``is_object`` returns false for it is a first WTF, but not the only one)